### DISCUSSION TITLE
CAN BUS BUFFER

### DISCUSSION THREAD

**Discussion by KBS91WAW** [sandeepmistry/arduino-CAN#95] [Q&A]
Hello Everyone,

I worked in past with PIC, DSPIC and STM32. On every of them I was able to calibrate finite number
of buffers that were related with some CAN Ids I was looking at. 
Is that possible here?

Regards,
Chris

**Comment by timurrrr** [comment]
ESP or MCP?
My understanding of the MCP datasheet is that it doesn't have hardware buffers, only two sets of registers (effectively two "buffered" messages at most).
